lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -69,3 +69,5 @@ ecdsa==0.13
 # Flexible BAM index naming
 pysam==0.8.4+gx1
 
+# CSG dependencies
+numpy-stl==1.8.0
```

Comment:
We should have now a wheel for version 1.9.1
https://github.com/galaxyproject/starforge/pull/104
Can you adjust this and then we are ready to go.